packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-11/zoe-upgrade/vat-status.mjs

```diff
@@ -0,0 +1,105 @@
+// @ts-check
+import processAmbient from 'process';
+import dbOpenAmbient from 'better-sqlite3';
+
+/**
+ * @file look up vat incarnation from kernel DB
```

Comment:
@warner does it seem like a good idea for a test to do this "clean room" style implementation of querying the kernel DB?

cc @raphdev